packages/core/strapi/lib/types/core/common/controller.d.ts

```diff
@@ -1,8 +1,8 @@
-import type koa from 'koa';
+import type { ExtendableContext, Next } from 'koa';
 
 export type ControllerHandler = <T>(
```

Comment:
This `ControllerHandler` is used to define the type below in `Controller` But the Generic is ignored